### ISSUE TITLE
error when generating deis-dev

### ISSUE THREAD

**Issue by bacongobbler** [deis/charts#125]
Updated to latest chart, using helm 0.4.0:

```
><> rm -rf ~/.helm/workspace/
><> helm fetch deis-dev
---> Fetched chart into workspace /home/bacongobbler/.helm/workspace/charts/deis-dev
---> Done
><> helm generate deis-dev
[ERROR] Error opening value file: Near line 45 (last key parsed 'gcs.key_json'): Expected value but found '`' instead.
[ERROR] Failed to complete generation: failed to execute helm template -o /home/bacongobbler/.helm/workspace/charts/deis-dev/manifests/deis-registry-rc.yaml -d /home/bacongobbler/.helm/workspace/charts/deis-dev/tpl/objectstorage.toml /home/bacongobbler/.helm/workspace/charts/deis-dev/tpl/deis-registry-rc.yaml (/home/bacongobbler/.helm/workspace/charts/deis-dev/tpl/deis-registry-rc.yaml): exit status 1
```

**Comment by bacongobbler** [comment]
changing the following in the chart resolved this for me:

```
[gcs]                                                                                              
# key_json is expanded into a JSON file on the remote server. It must be                           
# well-formatted JSON data.                                                                        
key_json = ""                                                                                      
bucket = "your-bucket-name"
```

Changing the three backtick syntax for `key_json` to quoted strings worked for me.

**Comment by helgi** [comment]
Isn't this fixed by #124

**Comment by mboersma** [comment]
> Isn't this fixed by #124 

Yes indeed, so @bacongobbler `helm up` should get things going again for you. Please re-open this issue if I'm wrong.
